README.md

```diff
@@ -60,6 +60,15 @@ export PATH=$PATH:$HOME/.local/bin
 poetry install
 ```
 
+### PlantUML example
+Files of a single type can be "translated" to PlantUML diagrams. See the following example:
```

Comment:
supernit: consider putting a newline here 😛.

Reply:
Ok, I'll make this change